chaos_kitten/toys_cli.py

```
@@ -0,0 +1,225 @@
+                    dest_path.unlink()
+                temp_path.rename(dest_path)
+                
+                console.print(f"[green]Successfully installed '{slug}' to {dest_path}[/green]")
+            else:
+                console.print(f"[red]Failed to download profile: HTTP {response.status}[/red]")
+    except Exception as e:
+        console.print(f"[red]Error downloading profile: {e}[/red]")
+
+@toys_app.command("list")
+def list_toys(
+    installed: bool = typer.Option(True, "--installed", help="List installed profiles"),
```

Ruff (0.15.1)

[warning] Unused function argument: `installed`

(ARG001)